pupa/tests/importers/test_membership_importer.py

```diff
@@ -89,3 +89,40 @@ def test_no_membership_for_person_including_party():
 
     with pytest.raises(NoMembershipsError):
         memimp.import_data([p._related[0].as_dict()])
+
+
+@pytest.mark.django_db
+def test_multiple_orgs_of_same_class():
+    """
```

Comment:
I'm not sure that this comment applies to this test. Seems to have been copied from `test_no_membership_for_person_including_party`.